<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(149, 106, 214);
font: 87 22pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>CẬP NHẬT NHÀ CUNG CẤP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_luu">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>LƯU</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_tenNCC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>SỐ ĐIỆN THOẠI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>ĐỊA CHỈ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>TÊN NHÀ CUNG CẤP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_diachi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_sdt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_maNCC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>MÃ NHÀ CUNG CẤP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>GHI CHÚ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_note">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
